zaas-service/src/main/resources/gateway-log-messages.yml

```diff
@@ -10,7 +10,7 @@ messages:
 
     # TLS,Certificate messages
     # 500-599
-    - key: org.zowe.apiml.gateway.security.x509.missingCertificate
```

Comment:
I put the comment here because I cannot on the file itself. This file should be renamed as zaas-log-messages

Reply:
I am working on in the next PR